.circleci/config.yml

```diff
@@ -66,9 +66,14 @@ jobs:
       TEST_DIR: "controller/integration/test/reconciliation"
     <<: *e2eTest
 
-  e2eTestInformerIntegration:
+  e2eTestInformerIntegrationBasic:
     environment:
-      TEST_DIR: "informer/integration"
+      TEST_DIR: "informer/integration/test/basic"
```

Comment:
I moved the existing test and aligned it with the new folder structure which enables us to have multiple tests like we have in other projects.